docs/tutorials/connecting-to-the-internet-via-wifi.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Setting up wifi on ev3
+subject: Networking
+---
+
```

Comment:
You should say that on the EV3, you will have to connect via ssh some other way first (USB or Bluetooth) but on the RPi, you can attach a monitor and keyboard directly.